web/src/main/java/no/nav/foreldrepenger/tilganger/InnloggetNavAnsattDto.java

```diff
@@ -1,15 +1,17 @@
 package no.nav.foreldrepenger.tilganger;
 
+import jakarta.validation.constraints.NotNull;
+
 import java.time.LocalDateTime;
 import java.util.Objects;
 
-public record InnloggetNavAnsattDto(String brukernavn,
-                                    String navn,
-                                    boolean kanSaksbehandle,
-                                    boolean kanVeilede,
-                                    boolean kanOverstyre,
-                                    boolean kanOppgavestyre,
-                                    boolean kanBehandleKode6,
+public record InnloggetNavAnsattDto(@NotNull String brukernavn,
+                                    @NotNull String navn,
+                                    @NotNull boolean kanSaksbehandle,
```

Comment:
Hvorfor NotNull på en skalar type ?

Reply:
OpenApi spec'en vil ikke generere properties som required med mindre de er annotert @NotNull, også for scalar-typer.